components/bio-formats-tools/src/loci/formats/tools/XMLValidate.java

```diff
@@ -47,17 +47,62 @@
  */
 public class XMLValidate {
 
-  public static void process(String label, BufferedReader in)
+  private static boolean processFile(String label, BufferedReader in)
```

Comment:
Should we make this `public` as the low-level method for validated a `BufferedReader` and returning an error code?

Also not very keen on `processFile` since there is no file at this stage. Thoughts about calling all the new public methods `public static <output> validate(<input>)` with different input arguments and return types